.github/actions/e2e/cleanup/action.yaml

```diff
@@ -42,6 +42,22 @@ runs:
               break
           fi
         done
+    - name: delete-cluster
+      shell: bash
+      run: |
+        eksctl delete cluster --name ${{ inputs.cluster_name }} --timeout 60m --wait || true
+     - name: delete-network-interfaces
+      shell: bash
+      run: |
+        aws ec2 describe-network-interfaces \
```

Comment:
Sanity check: does this work correctly if the output of this function is empty?

Reply:
Confirmed, works as expected if the function is an empty string